### ISSUE TITLE
training schedule regarding unsupervised/supervised loss 

### ISSUE THREAD

**Issue by dvorak0** [simonmeister/UnFlow#4]
What if I train on synthetic dataset using supervised loss first, and then fine tune on real-world dataset using unsupervised loss? Would the performance be competitive?

Thanks.

**Comment by simonmeister** [comment]
Although we did not try this, i suppose you should achieve at least as good results as with an unsupervised loss for pre-training. If you have dense flow ground truth, the supervised loss from this should always be more accurate than the approximate unsupervised loss.
